src/components/MarkdownPage/MarkdownPage.js

```diff
@@ -19,8 +19,20 @@ import toCommaSeparatedList from 'utils/toCommaSeparatedList';
 import {sharedStyles} from 'theme';
 import createOgUrl from 'utils/createOgUrl';
 
+type Props = {
+  authors: Array<string>,
+  createLink: Function,
```

Comment:
Would be good to add a TODO here to better type this (once we Flow-type `createLink` and the `Section` component)